tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -2,11 +2,16 @@
 # See LICENSE.txt for details.
 
 # Executable: EvolveGhKerrSchild
-# Check: parse;execute
+# Check: parse;execute_check_output
 # Timeout: 8
 # ExpectedOutput:
 #   GhKerrSchildVolume0.h5
 #   GhKerrSchildReductions.h5
+# OutputFileChecks:
+#   - Label: "check_news"
+#     Subfile: "/AhA.dat"
+#     FileGlob: "GhKerrSchildReductions.h5"
+#     AbsoluteTolerance: 1e2
```

Comment:
The first column is time and the second is areal mass. I think it's better to skip at least the first column, maybe even both.

Reply:
I'm not sure I see the point in skipping any of the columns, really... The tolerance needs to be high to accommodate the area value, and all this is doing is verifying existence, so I don't think it makes any difference for that check to exclude columns. I'd feel differently if this was performing an actual comparison to an expected value.

Reply:
ok